src/main/resources/templates/slurm/command/sinfo

```diff
@@ -0,0 +1,5 @@
+sinfo
+[# th:if="${partitionName != null}"]
+    --partition=[(${partitionName})]
+[/]
+-h -o %all
```

Comment:
may be we could make some our own format with only required fields instead of `%all` usage for command result output and use MapStruct for result parsing

[https://slurm.schedmd.com/sinfo.html#OPT_format](https://slurm.schedmd.com/sinfo.html#OPT_format)

Reply:
fixed